src/platform/platform_posix.c

```diff
@@ -665,17 +665,23 @@ CxPlatThreadCreate(
 #else // CXPLAT_USE_CUSTOM_THREAD_CONTEXT
 
     //
-    // If pthread_create fails with ENOKEY or ENOENT, then try again without the attribute
+    // If pthread_create fails with an error code, then try again without the attribute
     // because the CPU might be offline.
     //
-    if (pthread_create(Thread, &Attr, Config->Callback, Config->Context) &&
-        ((errno != ENOKEY && errno != ENOENT) || pthread_create(Thread, NULL, Config->Callback, Config->Context))) {
-        Status = errno;
+    if (pthread_create(Thread, &Attr, Config->Callback, Config->Context)) {
         QuicTraceEvent(
             LibraryErrorStatus,
-            "[ lib] ERROR, %u, %s.",
-            Status,
-            "pthread_create failed");
+            "[ lib] WARNING, %u, %s.",
+            errno,
+            "pthread_create failed, retrying without affinitization");
```

Comment:
I have a feeling this requires rerunning some tool to update some autogenerated files, what do I need to do? @nibanks?

Reply:
```c
                QuicTraceLogWarning(
                    PlatformThreadCreateFailed,
                    "[ lib] pthread_create failed, retrying without affinitization");
```
Then run `.\scripts\update-sidecar.ps1`.